deployment/ansible/roles/cac-tripplanner.otp-data/tasks/main.yml

```diff
@@ -5,6 +5,11 @@
 - name: Copy Feed Validator
   copy: src=validate_feed.py dest="{{ otp_data_dir }}"
 
+- name: Download OTP Data
+  command: aws s3 sync s3://cleanair-otp-data "{{ otp_data_dir }}/"
+  sudo: no
+  when: production or test
```

Comment:
What's the reason for not also downloading the files in development? Seems like they may come in handy, at least for running local tests.